.github/workflows/pr_agent.yml

```
@@ -17,4 +17,8 @@ jobs:
         uses: Codium-ai/pr-agent@main
         env:
           OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
-          GITHUB_TOKEN: ${{ secrets.QODO_MERGE_GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.QODO_MERGE_GITHUB_TOKEN }}
+          pr_description.generate_ai_title: false
+          pr_description.extra_instructions: "(MUST)回答は日本語で記述してください。"
+          pr_reviewer.extra_instructions: "(MUST)回答は日本語で記述してください。"
+          pr_reviewer.require_score_review: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)